utils/nctl/sh/scenarios-upgrades/upgrade_scenario_06.sh

```diff
@@ -284,46 +286,60 @@ function _step_10()
     fi
 }
 
-# Step 11: Assert lfbs are in sync
+# Step 11: Check reactor state
```

Comment:
Looks like the code used in `function _step_11()` is repeated in other tests. Did you consider extracting it to some utility file?

Reply:
discussed in slack, leaving this as is for now